### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by tamird**
Chores courtesy of Docker, as usual.

@paperstreet @tschottdorf for the docker changes

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/6405)

### CODE REVIEW

File: acceptance/cluster/docker.go [hunk 1]
```diff
@@ -97,19 +97,19 @@ func hasImage(l *LocalCluster, name, tag string) bool {
 	return false
 }
 
-func pullImage(l *LocalCluster, options types.ImagePullOptions) error {
+func pullImage(l *LocalCluster, ref string, options types.ImagePullOptions) error {
 	// Hack: on CircleCI, docker pulls the image on the first access from an
```
Comment (tschottdorf) [inline acceptance/cluster/docker.go]: Any chance this isn't an issue any more now?

**Comment by tschottdorf** [comment]
> Nice API you got there. Would be a shame if .... something happened to it

LGTM

File: acceptance/cluster/docker.go (hunk 1 above)
Comment (tamird) [inline acceptance/cluster/docker.go]: i doubt it. this isn't a docker version upgrade, only a change in the client library.

**Comment by tamird** [comment]
Accidentally lost jemalloc, returned now.

**Comment by paperstreet** [comment]
:lgtm:

---

Review status: 0 of 9 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/6405#-:-KGX5WeAdT8TlLpzHEKO:1417088721)*

**Pull request merged**